contentwarehouse/snippets/update_document_sample.py

```diff
@@ -25,8 +25,10 @@ def sample_update_document(
     """Updates a document.
```

Comment:
Do the same on this file for changing 

```
from google.cloud import contentwarehouse_v1 as contentwarehouse
```

to 
```
from google.cloud import contentwarehouse
```

And change `contentwarehouse.types.Document` → `contentwarehouse.Document`
`contentwarehouse.types.CreateDocumentResponse` → `contentwarehouse.CreateDocumentResponse`